plugins/outdated.yaml

```diff
@@ -43,6 +43,14 @@ spec:
     bin: outdated.exe
   shortDescription: Finds outdated container images running in a cluster
   homepage: https://github.com/replicatedhq/outdated
+  caveats: |
+    Usage:
+      $ kubectl outdated
```

Comment:
Basic usage instructions are shown by default in krew. So please consider dropping thsi.